src/test/scala/uk/gov/nationalarchives/LambdaTest.scala

```diff
@@ -123,15 +137,16 @@ class LambdaTest extends AnyFlatSpec with BeforeAndAfterEach {
 
     metadataFiles.foreach { file =>
       s3Server.stubFor(
-        put(urlEqualTo(s"/$testOutputBucket/$reference/$file"))
-          .willReturn(ok())
+        put(urlEqualTo(s"/$testOutputBucket/$reference/${file._1}"))
+          .willReturn(ok().withHeader("x-amz-checksum-sha256", convertChecksumToS3Format(file._2)))
       )
     }
 
-    uuids.foreach { uuid =>
+    uuids.foreach { uuidAndChecksum =>
```

Comment:
If you add `case (uuid, checksum)` then you wouldn't need the `val uuid` line

Reply:
That's a lot neater, cheers.